src/store.js

```diff
@@ -0,0 +1,61 @@
+// store.js
+import React, { createContext, useReducer } from "react";
+import camera from "./camera.png";
+
+const initialState = {
+  newVehicle: {
+    step: 1,
+    question: "Do you have your vehicle's registration card?",
+    notification: false,
+    btn1: {
+      small: true,
+      text: "Yes",
+      image: null,
+      imageOptions: null,
+      action: "newVehicle notify",
+    },
+    btn2: {
+      small: true,
+      text: "No",
+      image: null,
+      imageOptions: null,
+    },
+  },
+};
+const store = createContext(initialState);
+const { Provider } = store;
+
+const StateProvider = ({ children }) => {
```

Comment:
this looks great! will need to be briefed for the reasoning in using switch case.